source/solvers/gls_sharp_navier_stokes.cc

```diff
@@ -470,8 +477,16 @@ GLSSharpNavierStokesSolver<dim>::force_on_ib()
                                           fluid_pressure[d][d] =
                                             pressure_values[k];
                                         }
+                                      shear_rate =
+                                        velocity_gradients[k] +
+                                        transpose(velocity_gradients[k]);
+                                      viscosity =
+                                        rheological_model->get_viscosity(
+                                          rheological_model
+                                            ->get_shear_rate_magnitude(
+                                              shear_rate));
                                       fluid_stress =
-                                        mu *
+                                        viscosity *
                                           (velocity_gradients[k] +
                                            transpose(velocity_gradients[k])) -
```

Comment:
We could reuse the shear_rate tensor defined previously defined

Reply:
Yes I forgot! thank you